include/zephyr/drivers/rtc/maxim_ds3231.h

```diff
@@ -15,7 +15,7 @@
  *
  * The core Zephyr API to this device is as a counter, with the
  * following limitations:
- * * counter_read() and counter_*_alarm() cannot be invoked from
+ * * ``counter_read()`` and ``counter_*_alarm()`` cannot be invoked from
```

Comment:
These are both APIs that don't exist anymore so it'd be nice to update this part, I guess. Maybe @pabigot can suggest an update as the original author?